### ISSUE TITLE
Issue: LivePortrait looking for .pth file instead of .safetensors

### ISSUE THREAD

**Issue by amirrrr234** [kijai/ComfyUI-LivePortraitKJ#140]
Hi,

I'm encountering an issue with LivePortrait in ComfyUI. The system is attempting to load the appearance_feature_extractor.pth file, but I only have the appearance_feature_extractor.safetensors file. I have placed the .safetensors file in the correct directory (ComfyUI\models\liveportrait\base_models), but the script continues to look for the .pth file, resulting in the following error:

[Errno 2] No such file or directory: 'C:\\Users\\...\\ComfyUI\\models\\liveportrait\\base_models\\appearance_feature_extractor.pth'

Is there a way to either modify the script to use the .safetensors file or should I be downloading a .pth version of the model? If so, where can I find that?

Any help or guidance would be appreciated!

Thank you!

**Comment by kijai** [comment]
Are you sure you are actually using these nodes? I do not recognize that path in your error. These nodes load .safetensors from just `ComfyUI\models\liveportrait`

**Comment by amirrrr234** [comment]
Yes. I think the package i installed is from another author. Thanks a lot for your response.
